tech/tikui.ts

```diff
@@ -0,0 +1,42 @@
+import * as commander from 'commander';
+import * as concurrently from 'concurrently';
+import * as path from 'path';
+import * as rimraf from 'rimraf';
+
+const BUILD_DIR = path.resolve(__dirname, '..', 'build');
+
+const SASS_CACHE = 'sass src:cache -s expanded --watch';
+const EXPRESS_SERVE = `node "${path.resolve(BUILD_DIR, 'express.js')}"`;
+const SASS_BUILD = 'sass src:dist  -s compressed --source-map --embed-sources';
```

Comment:
You can remove extra space `sass src:dist -s compressed --source-map --embed-sources`

Reply:
THis is now OK